### ISSUE TITLE
New Enhancment Ideas

### ISSUE THREAD

**Issue by LutraVibes** [rafaelvcaetano/melonDS-android#1260]
Hello! 👋

I have a few new enhancment ideas for MelonDS, and here they are:

1. Standard Layout Size Button - Add a button by the scaling bar (when editing a layout) that sets the scale of the selected button back to the default; 104
2. Themes - Create a theme swapper with colour themes similar to Dolphin, along with custom colour selection similar to Chrome's, where you can select the main and accent colours from a preset table
3. Emulation Colour Plates - Swappable colour presets, similar to swappable plates, that create shading/shadows on the main colour buttons and the accent colour for the background, with a colour table similar to what was said previously like in Chrome (this would affect emulation background and buttons, not home page)
4. Stylized Home Menu - DSi-like game selection menu, with a separate DSiWare section (to show they cannot be chosen just from there, but rather booted via firmware) with a nostalgic pixelated feel, and options inside a (rounded rectangular) floating tab at the bottom of the screen (video settings, audio settings, cheats, etc.) that would have shadows to give a 3D feel
5. Super XBR Scaler - Add the Super XBR scaling filter to the video options (removes jagged lines and makes everything MUCH smoother, unlike Quilez and XBRZ)
6. App Intro - Introduce a DS/DSi-like intro to the app that says "Melon" and then the little "DS" pops up on the corner of it, just like the DSi
7. Circle Pad - Add an option under controls to replace the D-Pad with a circle pad for easier play (let's be honest, nearly nobody likes the D-Pad when it is on a touch screen)

I appreciate your consideration of these ideas, and if you are already working on these or have already thought of these enhancements. Keep up your amazing work. 👍

**Comment by LutraVibes** [comment]
Three months...

**Comment by WinUser-Generic** [comment]
I have other suggestion to add to point 5.
Add increment internal resolution x1 x2 x3 x4 like most of emulators have to get a better resolution, and antialiasing and anisotropic filter options to would be nice to try to clean 3d most of we can.

**Comment by LutraVibes** [comment]
> I have other suggestion to add to point 5. Add increment internal resolution x1 x2 x3 x4 like most of emulators have to get a better resolution, and antialiasing and anisotropic filter options to would be nice to try to get clean 3d most of we can.

Added to the suggestion list.

**Comment by LutraVibes** [comment]
@rafaelvcaetano please take a look at the new suggestions if you haven't already.

**Comment by LutraVibes** [comment]
@rafaelvcaetano please take a look at the new suggestions if you haven't already. 💯

**Comment by vvvvvvitor** [comment]
Wouldn't these be better as separate issues?

**Comment by LutraVibes** [comment]
> Wouldn't these be better as separate issues?

Not really. I would then end up with 13 different issues. Not preferable.
